src/index.html

```diff
@@ -7,8 +7,87 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo__link">
+        <img src="./images/logo.png" alt="logo-img">
+      </a>
+    
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-value">
+            <a 
+              class="nav-link is-active" 
+              href="#Apple"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              class="nav-link" 
+              href="#Samsung"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              class="nav-link" 
+              href="#Smartphones"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              data-qa="hover"
+              class="nav-link" 
+              href="#L&c"
+            >
+            laptops & computers
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              class="nav-link"
+              href="#Gadgets"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              class="nav-link" 
+              href="#Tablets"
+            >
+              tablets
+
+            </a>
+          </li>
+          <li class="nav-value">
+            <a 
+              class="nav-link" 
+              href="#Photo"
+            >
+              photo
+
+            </a>
+          </li>
```

Comment:
```suggestion
            >
              photo
            </a>
          </li>
```
why didn't you fix it?